spec/system/consumer/account/developer_settings_spec.rb

```diff
@@ -64,4 +79,9 @@
       end
     end
   end
+
+  def second_table_line
+    # It's actually the third line when you include the header
+    '(//tr)[3]'
+  end
```

Comment:
We could also select the line based on the text "payment status", similar to the admin products test  https://github.com/openfoodfoundation/openfoodnetwork/blob/ded139458d3d684506d1f6868d502cf4b97e94c3/spec/support/products_helper.rb#L42-L47

Then it doesn't matter what order the rows are in.

Reply:
I think I was looking at this the wrong way, I just discovered you can use the `:table_row` selector : https://rubydoc.info/gems/capybara/3.39.2/Capybara/Selector
Much cleaner than what I did.